Jenkinsfile

```diff
@@ -1332,17 +1099,18 @@ pipeline {
                     }
                     agent {
                         // 2 node cluster with 1 IB/node + 1 test control node
-                        label 'ci_nvme3'
+                        label 'ci_nvme3x'
```

Comment:
I assume this shouldn't land, so just noting it.

Reply:
Yeah, I missed putting that back after I did a test of that build agent.
I figured that there would be a few other changes.